### ISSUE TITLE
iOS app crashes launching Firebase.init();

### ISSUE THREAD

**Issue by PippoApps** [myflashlab/Firebase-ANE#189]
## Environment ##
* iOS Version: 9.3.5
* AIR SDK Version: 28.0.0.125
* Latest version of ANEs setup with AneLab 1.1.11

## Problem Description ##
I am configuring everything with AneLab, works perfectly on Android. On iOS though, once app is launched, it crashes as soon as Firebase.init(true) is called.

On iOS console I get this error:
Feb 23 20:25:38 Filippos-iPad BuncoBonko_Device[1578] <Error>: *** Terminating app due to uncaught exception ‘com.firebase.appinvite’, reason: ‘Firebase Invites has stopped your project due to a misconfiguration in your project that may prevent your app from behaving as expected:
    
Error: URL scheme needed for Google Sign-In not included in your app’s Info.plist. Please refer to the Google Sign-In SDK documentation for more information.’

I couldn't find any instruction on your wiki about this. Can you point me in the right direction?
thanks

**Comment by PippoApps** [comment]
![image](https://user-images.githubusercontent.com/29799572/36601375-47004a2a-18b5-11e8-92fc-bc7b1e137af0.png)
This is the version of ANEs I am using (consider in this list you will see also Andorid ANEs, which are not included in the iOS project).

**Comment by PippoApps** [comment]
Double checked, all frameworks are there, and bundles are included.
![image](https://user-images.githubusercontent.com/29799572/36605698-bd304c70-18c1-11e8-8ec1-0c28c0e2fd20.png)

**Comment by PippoApps** [comment]
I had missed that, it's explained here that provisionings need to be re-exported after adding Dynamic Links:
https://github.com/myflashlab/Firebase-ANE/wiki/I.2-Init-Dynamic-links
I am closing this issue till I try with new provisionings.
cheers

**Comment by PippoApps** [comment]
![image](https://user-images.githubusercontent.com/29799572/36610329-30205078-18d0-11e8-8992-ca8e930dea2b.png)
ANEs as setup by AneLab

**Comment by PippoApps** [comment]
As stated here: https://stackoverflow.com/questions/41002171/how-to-disable-google-sign-in-coming-with-firebase-pods
The value to insert when embedding invites ane is the "reversed client id", not client id, as stated in wiki and in help rollover of AneLab. 
![image](https://user-images.githubusercontent.com/29799572/36621948-06379188-18fb-11e8-8304-6ad159b9256c.png)

**Comment by myflashlab** [comment]
Correct, it should be the reversed client ID. thanks for finding these small mistakes in the descriptions. Anyhow, about the error message, that happens if you are missing the following in your manifest:

```xml
<key>CFBundleURLTypes</key>
  <array>
    <dict>
      <key>CFBundleURLSchemes</key>
      <array>
        <string>com.googleusercontent.apps.16273636234-5j7j3ara6qgafjub1229im7h2q8lgceb</string>
      </array>
      <key>CFBundleTypeRole</key>
      <string>Editor</string>
      <key>CFBundleURLName</key>
      <string>google</string>
    </dict>
  </array>
```
check if your manifest has this part?

**Comment by myflashlab** [comment]
Could you do a little test? Can you remove the FB ANE and see if a dynamic link can open up your app or not?

**Comment by PippoApps** [comment]
Yes dynamic link opens my app, even with FB ANE.
If I launch the app with Dynamic Link, app opens and event is fired correctly.
If I launch the app **without dynamic link**, event is never fired. On Android though, **event is fired anyway**, just without any data attached.
Is this a normal behavior? Should I differentiate Android and iOS?
On Android, I proceed initialization always after Dynamic Link event is fired, with or without link.
Should I proceed without listener on iOS, and just on next frame proceed initialization?
Thank you very much for your help.
